web/themes/joinup/templates/search_api/facets-type-wrapper--more.html.twig

```diff
@@ -12,8 +12,13 @@
  * @ingroup themeable
  */
 #}
+{% if items|length > 2 %}
+  {% set class='tab__container tab__container--wide' %}
```

Comment:
Hi @luke-waat I added the comment in the jira ticket, though here's a short summary here as well:

- 3 items to be shown, just as currently
- underline only active
- mobile - vertically stacked, centered
- all up - horizontal align, centered, max-width so to be clickable easily